frontend/apps/app/next.config.ts

```diff
@@ -102,6 +102,11 @@ const nextConfig: NextConfig = {
       },
     })
 
+    config.module.rules.push({
+      test: /\.mp4$/,
+      type: 'asset/resource',
+    })
```

Comment:
To import mp4 video file, I added these lines.

It used to be very common to use [`file-loader`](https://www.npmjs.com/package/file-loader/v/1.1.1), but the webpack itself supports the feature since webpack 5.

ref: https://webpack.js.org/guides/asset-modules/#resource-assets